### ISSUE TITLE
unable to validate csrf token

### ISSUE THREAD

**Issue by ekalaya2015** [minitriga/silverpeak_python#43]
hello,
first of all, I want to thank you for this great library.
Unfortunately, every time I invoke method after successful login, It gave me 401 response and response text said 'Unable to validate CSRF token'

My orchestrator version is 8.10

It would be great if you can help me with this problem

**Comment by minitriga** [comment]
Hi, I'm not able to recreate this issue on my orchestrator running 8.10.13. Could you please send me the output? and the code you are trying to run?

**Comment by ekalaya2015** [comment]
Hello,
Thank you for your response.
Indeed, if you use this library and invoke any methods after successful login, you won't get any responses like I did.
In my cases, there are some registered devices in orchestrator. If I use swagger, for example, API endpoint /appliance would give some results. but nothing for library usage. So I investigated the library. You can see the investigation screenshot below.
![image](https://user-images.githubusercontent.com/27433888/118638242-7834de80-b78b-11eb-9138-203eb8af4540.png)

login status (swagger)
![image](https://user-images.githubusercontent.com/27433888/118639086-5d169e80-b78c-11eb-86d0-903e49760547.png)

appliances list (swagger)
![image](https://user-images.githubusercontent.com/27433888/118639552-e4641200-b78c-11eb-9fd1-5e24aed22a40.png)


So, generally speaking there is a difference result between library and swagger.

Honestly, I am new in silver-peak sdwan. I haven't yet explored too much. But would this problem caused by orchestrator advance setting which is 'enforceCSRFCheck' (see below)?
![image](https://user-images.githubusercontent.com/27433888/118640392-c34ff100-b78d-11eb-9de5-f705f4f86b7d.png)
Btw, this orchestrator is our client's. I have limited access. I can not change any settings. 

Thank you

**Comment by minitriga** [comment]
Yes we have come across this problem with one of our customers and different monitoring solutions and we opened a ticket with Silver Peak and their recommendation is to disable the csrfProtection. This can be done by going to `Orchestrator -> Advanced Properties -> turn csrfProtection to false`. 
![image](https://user-images.githubusercontent.com/26367336/118713692-24bda180-b81a-11eb-899b-fa477a86ce28.png)

I will look into it again to see if we can incorporate a work around into the python client. We use it almost every day for doing different bits we released a new version today.
